component/service/src/main/java/org/exoplatform/social/rest/impl/space/SpaceRestResourcesV1.java

```diff
@@ -502,6 +509,19 @@ public Response getSpaceAvatarById(@Context UriInfo uriInfo,
         InputStream stream = identityManager.getAvatarInputStream(identity);
         if (stream != null) {
           builder = Response.ok(stream, "image/png");
+          int[] dimension = Utils.parseDimension(size);
+          try {
+            byte[] avatarContent = imageThumbnailService.getOrCreateThumbnail(identityManager.getAvatarFile(identity),
+                            identity,
+                            dimension[0],
+                            dimension[1])
```

Comment:
by using Utils.parseDimension, dimension[0] and dimension[1] can be empty

Reply:
Yes it can be 0 height and 0 width and this case is treated in the imageResizeService and the ImageThumbnailService